build_msvc/bitcoin_config.h

```diff
@@ -38,6 +38,12 @@
 /* Define to 1 to enable wallet functions */
 #define ENABLE_WALLET 1
 
+/* Define to 1 to enable BDB wallet */
+#define USE_BDB 1
+
+/* Define to 1 to enable SQLite wallet */
+#define USE_SQLITE 1
```

Comment:
Should we be adding this for 0.21?

Reply:
> Should we be adding this for 0.21?

No opinion (seems fine, not necessary), but `#define USE_SQLITE 1` is part of base PR #20202. @achow101, PR description could be clearer and say ***This is based on #20202.*** or something along those lines since it seems like I'm not the only person to not realize this PR contains changes from another PR